lib/global-styles.php

```diff
@@ -563,8 +563,9 @@ function gutenberg_experimental_global_styles_get_block_data() {
 				'global',
 				array(
 					'supports' => array(
-						'__experimentalFontAppearance' => false,
 						'__experimentalFontFamily'     => true,
+						'__experimentalFontStyle'      => false,
```

Comment:
We have a font family global style. Would it make sense to have font style or font weight global pickers?
I guess the font-weight may make sense at the global level. The font style I'm not sure. Any thoughts on that @jasmussen, @aristath, @shaunandrews?

Reply:
I think it would make sense to have a global for the font-weight... Depending on the font-family selected we may want to select a different font-weight globally, and inside the `theme.json` file use the css-var as default value. It would be a lot easier for a designer to just set the global font-weight to `300` instead of having to go in every block-type and make adjustments.
Font-style I'd say is not that necessary, but if I'm not mistaken font-style and font-weight were recently merged to a single control so we don't have much of a choice and it's all or nothing?

Reply:
The technical underpinnings are one thing, the user experience is another. While CSS suggests one thing, an italic font is a variant of the font family, not a boolean toggle, that's why [most apps that let you choose font weight](https://cloudup.com/cAtmxUMeVmV) create a unified dropdown — bold, bold italic, heavy, heavy italic, etc. And it's why it should be a single dropdown in this interface as well: italic is a visual font choice, not a toggle. 

It's fine for them to be separate when only one is enabled.